api/cpp/cmake/SlintConfig.cmake.in

```diff
@@ -14,8 +14,41 @@ endif()
 add_library(@slint_cpp_impl@ @cmake_lib_type@ IMPORTED)
 set_target_properties(@slint_cpp_impl@ PROPERTIES @SLINT_LIB_PROPERTIES@)
 
-set(SLINT_COMPILER @SLINT_COMPILER@ CACHE FILEPATH "Path to the slint-compiler executable")
+function(_slint_download_compiler_and_cache)
+    set(SLINT_GITHUB_RELEASE @PROJECT_VERSION@ CACHE STRING "GitHub Release to use for Slint Binary Artifact Downloads")
```

Comment:
does it need to be a CACHE variable?
It seems like it wouldn't be wise to change that as it needs to match exactly with the version of this package.

Reply:
There's one situation where the override is necessary: When using a pre-release. Then the release is `nightly` but the Slint version is (for example) `1.8.0`.